app/models/mixins/has_monitoring_manager_mixin.rb

```diff
@@ -0,0 +1,29 @@
+module HasMonitoringManagerMixin
+  extend ActiveSupport::Concern
+
+  included do
+    # TODO: why is this relation needed ? why is the inheritors relation no enough?
```

Comment:
I don't want anything inherited in any case. I was wondering why other secondary managers (NetworkManager I think) had this in the parent class. Why shouldn't I remove this relation and just have the relation in the base class?
cc @durandom